drivers/can/can_common.c

```diff
@@ -11,6 +11,9 @@
 #include <logging/log.h>
 LOG_MODULE_REGISTER(can_driver);
 
+#define CAN_CLAMP(val, min, max) MIN(MAX(val, min), max)
```

Comment:
This looks fairly generic, maybe that'd be nice to have in `util.h`?

Reply:
Just seeing PR https://github.com/zephyrproject-rtos/zephyr/pull/29576, which should be used once merged.

Reply:
Sure. Didn't knew that it is already in! Thanks for the hint.